eval/compute_qrel_statistics.py

```diff
@@ -0,0 +1,108 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Compute various statistics for qrels."""
+
+import argparse
+import collections
+
+
+def is_relevant(grade: int, threshold: int):
+    if grade > threshold:
```

Comment:
```suggestion
    return grade > threshold
```

Reply:
For future reference - What's the rationale for this? Is this a general style tip?

Reply:
No, it is just that you are replacing 4 lines of code with a single line.

I think it is easier to read `return grade > threshold` but that is only my option.